clients/python/docs/BasicTutorial_v0.6.0.ipynb

```diff
@@ -0,0 +1,532 @@
+    "    users_api = UsersApi(api_client)\n",
+    "\n",
+    "    profile = users_api.get_my_profile()\n",
+    "    print(profile)\n",
+    "\n",
+    "    #\n",
+    "    #  {'first_name': 'foo',\n",
+    "    #  'gravatar_id': 'aa33fssec77ea434c2ea4fb92d0fd379e',\n",
+    "    #  'groups': {'all': {'description': 'all users',\n",
+    "    #                     'gid': '1',\n",
+    "    #                     'label': 'Everyone'},\n",
+    "    #             'me': {'description': 'primary group',\n",
+    "    #                    'gid': '2',\n",
+    "    #                    'label': 'foo'},\n",
+    "    #             'organizations': []},\n",
+    "    #  'last_name': '',\n",
+    "    #  'login': 'foo@itis.swiss',\n",
+    "    #  'role': 'USER'}\n",
+    "    #"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "6912889e",
+   "metadata": {},
+   "source": [
+    "## Solvers Workflow\n",
+    "\n",
+    "The osparc API can be used to execute any computational service published in the platform. This means that any computational service listed in the UI under the [Services Tab](https://docs.osparc.io/#/docs/platform_introduction/services) is accessible from the API. Note that computational services are denoted as *solvers* in the API for convenience, but they refer to the same concept.\n",
+    "\n",
+    "\n",
+    "Let's use the sleepers computational service to illustrate a typical workflow. The sleepers computational service is a very basic service that simply waits (i.e. *sleeps*) a given time before producing some outputs. It takes as input one natural number, an optional text file input that contains another natural number and a boolean in the form of a checkbox. It also provides two outputs: one natural number and a file containing a single natural number."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "398bcd09",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import time\n",
+    "from pathlib import Path\n",
+    "from zipfile import ZipFile\n",
+    "from tempfile import TemporaryDirectory\n",
+    "\n",
+    "import osparc\n",
+    "\n",
+    "CLIENT_VERSION = tuple(map(int, osparc.__version__.split(\".\")))\n",
+    "assert CLIENT_VERSION >= (0, 4, 3)\n",
+    "\n",
+    "Path(\"file_with_number.txt\").write_text(\"3\")\n",
+    "\n",
+    "with osparc.ApiClient(cfg) as api_client:\n",
+    "\n",
+    "    files_api = osparc.FilesApi(api_client)\n",
+    "    input_file: osparc.File = files_api.upload_file(file=\"file_with_number.txt\")\n",
+    "\n",
+    "    solver_key: str = \"simcore/services/comp/itis/sleeper\"\n",
+    "    solver_version: str = \"2.0.2\"\n",
+    "    \n",
+    "    solvers_api = osparc.SolversApi(api_client)\n",
+    "    solver: osparc.Solver = solvers_api.get_solver_release(solver_key=solver_key, version=solver_version)\n",
+    "\n",
+    "    solver_ports = solvers_api.list_solver_ports(solver.id, solver.version)\n",
```

Comment:
@pcrespov all I do to document the new ports listing endpoint is to print them here in the new version of the basic tutorial. Did you have more in mind for documenting this?